topojson/ops.py

```diff
@@ -108,6 +111,41 @@ def strtree_query_index(tree, arc):
     return tree_index
 
 
+def explode(segments):
+    if not isinstance(segments, list):
+        segments = [segments]
+    list_explode = [
+        list(geom.geoms) if hasattr(geom, "geoms") else [geom] for geom in segments
+    ]
+    return list(itertools.chain.from_iterable(list_explode))
+
+
+def extract_lines(geom: geometry.base.BaseGeometry) -> geometry.base.BaseGeometry:
+    if isinstance(geom, geometry.LineString):
+        return geom
+    elif isinstance(geom, geometry.MultiLineString):
+        return geom
+    elif isinstance(geom, geometry.GeometryCollection):
+        geoms = [
+            geom for geom in geom.geoms
+            if (
+                not geom.is_empty
+                and (
+                    isinstance(geom, geometry.LineString)
+                    or isinstance(geom, geometry.MultiLineString)
+                )
+            )
+        ]
+        if len(geoms) == 0:
+            return geometry.LineString()
+        elif len(geoms) == 0:
```

Comment:
Currently this `elif` is never entered, did you mean `len(gems) == 1`?
```suggestion
        elif len(geoms) == 1:
```

Reply:
oops... indeed!